.circleci/config.yml

```diff
@@ -16,12 +16,14 @@ jobs:
               sudo mkdir -p -m 777 $CIRCLE_ARTIFACTS
               chmod +x ./Tests/scripts/*
               chmod +x ./Tests/lastest_server_build_scripts/*
+              sudo pip install flake8
```

Comment:
We should install this as part of the docker image we use. We want a reproducible build. Also possible to use pipenv.

Reply:
Yes you are right, but until the docker image won't be ready lets add it this way, i'll try to make the docker image tomorrow